datafusion-cli/Cargo.lock

```diff
@@ -584,9 +584,9 @@ dependencies = [
 
 [[package]]
 name = "aws-sdk-sts"
-version = "1.41.0"
```

Comment:
If we execute `cargo update`, the version will be updated to `1.41.0` which isn't compatible with Rust 1.76, then the msrv check will fail.  I'm not sure if there is a better way to limit the version in the cargo toml file to avoid the `update` command updates an invalid version 🤔

Reply:
Yes I have actually worked around this on https://github.com/apache/datafusion/pull/12032#discussion_r1742677561